pkg/osquery/extension.go

```diff
@@ -229,6 +231,15 @@ func NodeKeyFromDB(db *bolt.DB) (string, error) {
 	}
 }
 
+func isGRPCInvalid(err error) bool {
```

Comment:
should you call this isNodeInvalidError(err)? using gRPCInvalid sounds like there's something wrong with the transport itself